src/main/java/reposense/git/GitClone.java

```diff
@@ -22,28 +23,38 @@
  * Git clone is responsible for cloning a local/remote repository into a new directory.
  */
 public class GitClone {
-
-    public static final String GIT_CLONE_BARE_OPTION = "--bare";
     private static final Logger logger = LogsManager.getLogger(GitClone.class);
 
     /**
-     * Clones a bare repo specified in the {@code repoConfig}.
+     * Runs {@code git clone --bare} command asynchronously to clone a bare repo specified in the {@code config}
```

Comment:
`@code` is to highlight code so shouldn't actually be used on string; we normally use it to highlight variables. Use a normal quotation mark or html `<q>` instead.